src/getCypressElementCoordinates.ts

```diff
@@ -10,7 +10,9 @@ export type Position =
   | "bottomRight"
   | { x: number; y: number };
 
-  function getPositionedCoordinates(
+export type ScrollBehaviorOptions = false | 'center' | 'top' | 'bottom' | 'nearest';
```

Comment:
This is actually defined in the Cypress namespace as of 6.1.0
https://github.com/cypress-io/cypress/blob/ff095df144042bac3bc8ccf611efac5096a47ab9/cli/types/cypress.d.ts#L2301

I'm not sure how to handle pulling in types from a peer dependency that might not have those types defined in the available version.

Reply:
It's ok to put it here, also we can get the type from Dom typings.